webknossos/tests/binaryData/Organization_X/e2006_knossos/datasource-properties.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
@fm3 Is it fair to assume that this dataset exists locally? It is a [part of the test db](https://github.com/scalableminds/webknossos/blob/master/test/db/dataSets.csv#L7), that's why I chose this name. It's needed for the annotation upload (and I think also the dataset-specific requests). Here, I just copied the small `testdata/simple_wk_dataset` to have a valid dataset.

Reply:
Hm, l4_sample is not visible for the test user `user_A@scalableminds.com`, I'm getting
```
AssertionError: Failed to upload annotation: 404: {"messages":[{"error":"Couldn't find the requested resource."}]}
```

 @fm3 Could you help me to adapt the test db that the dataset is visible?